JSDemos/Demos/DataGrid/FilterPanel/description.md

```diff
@@ -1 +1,7 @@
-This example shows how to&nbsp;filter data using the filter panel. You can use its check box to&nbsp;enable/disable the current filter expression, and clicking on&nbsp;this expression opens the integrated filter builder. Note that changes made in&nbsp;it&nbsp;are reflected in&nbsp;the filter row and header filter, and vice versa.
\ No newline at end of file
+[Filter panel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterPanel/) is a UI element that displays a combined filter. This filter is stored in the [filterValue](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterValue) option and consists of filters that users apply in other UI elemens ([filter row](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/filterRow/), [header filter](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/headerFilter/), [filterBuilder](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#filterBuilder)).
```

Comment:
I"d say "the combined filter", because it the only one - the one that combines all others.